src/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_json_debug_options.py

```diff
@@ -37,8 +38,13 @@ def to_json(self):
         return json.dumps(dct)
 
     def update_fom_debug_options(self, debug_options):
-        if 'DEBUG_STDLIB' in debug_options:
-            self.debug_stdlib = debug_options.get('DEBUG_STDLIB')
+
+        if 'JUST_MY_CODE' in debug_options:
```

Comment:
@int19h Do we need the debug options part at all?

Reply:
Nope.

Reply:
I've created an issue to address that:

https://github.com/microsoft/debugpy/issues/64